src/internal/connector/exchange/data_collections.go

```diff
@@ -205,9 +206,27 @@ func DataCollections(
 			continue
 		}
 
+		categories[scope.Category().PathType()] = struct{}{}
+
 		collections = append(collections, dcs...)
 	}
 
+	if len(collections) > 0 {
+		baseCols, err := graph.BaseCollections(
+			ctx,
+			acct.AzureTenantID,
+			user,
+			path.ExchangeService,
+			categories,
+			su,
+			errs)
+		if err != nil {
+			return collections, nil, err
```

Comment:
I'm a little hesitant returning collections here. It makes it feel like this is a "maybe we'll do this, maybe we won't" which makes enforcing invariants about it down the road kind of useless

Reply:
oh, yeah, totally with you.